packages/frontend/src/components/staking/components/Validators.js

```diff
@@ -1,15 +1,27 @@
-import React, { useState } from 'react';
+import React, { useState, useMemo } from 'react';
 import { Translate } from 'react-localize-redux';
 
+import { getProjectValidatorPrefix } from '../../../utils/constants';
+import { MAINNET, TESTNET } from '../../../utils/constants';
+import { wallet } from '../../../utils/wallet';
 import ListWrapper from './ListWrapper';
 import ValidatorBox from './ValidatorBox';
 
-export default function Validators({ validators, stakeFromAccount }) {
 
+export default function Validators({ validators, stakeFromAccount }) {
     const [validator, setValidator] = useState('');
-
+    const networkId = wallet.connection.provider.connection.url.indexOf(MAINNET) > -1 ? MAINNET : TESTNET;
+    const validatorPrefix = getProjectValidatorPrefix(networkId);
     const validValidator = validators.map(validator => validator.accountId).includes(validator);
 
+    const filterValidators = useMemo(() =>
```

Comment:
For now, we can't modify how we sort validators without a buy-in at the product level.  If I recall correctly, we intentionally _shuffle_ the list of validators every time we fetch them to avoid favoritism or 'gaming the system' by making all the validators start with the letter 'a' or other goofy things.  Since there is only 1 pool currently (AFAIK) implementing this new pool type, this would result in us putting that one pool at the top of the list every time, which we need to avoid to maintain our existing policy of randomizing the order of validators in the list.

Reply:
Restored initial rendering